projects/fal/pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "fal allows you to run python scripts directly from your dbt proje
 homepage = "https://github.com/fal-ai/fal"
 documentation = "https://docs.fal.ai"
 repository = "https://github.com/fal-ai/fal"
-readme = "README.md"
+readme = "../../README.md"
```

Comment:
Can you instead move the README inside of the fal folder? You can add a top-level readme just ponting to each project's README for now and we can improve it in next PRs.

Reply:
That's the idea, but I didn't want to change it now since the top-level readme has some important content to be on the first page. So I think the solution is to create `projects/fal/README.md` with more package-specific info and update the root README with links to the packages.

I'll tackle that as a next step. In the meantime I made this change so `pip install fal` and `pip install -e projects/fal` work.

Reply:
Awesome, OK! Let's make that one a fast follow-up then